style.css

```
@@ -0,0 +1,70 @@
+body {
+    background-color: rgb(253, 251, 251);
+
+}
+
+table , tr , td {
+    margin: 0;
+    border: 0;
+    padding: 0;
+}
+
+main {
+    width: 1366px;
+    background-color: lightgray;
+}
+
+#cabecalho {
+    background-color: rosybrown;
+    position: fixed;
+    top: 0;
+    width: 1366px;
+    height: 170px;
+}
+
+#titulo {
+    padding: 10px;
+}
+
+.menu {
+    display: inline-block;
+    width: 100px;
+    text-align: center;
+}
+
+#minha_foto {
+    width: 150px;
+    border-radius: 50%;
+    position: absolute;
+    left: 600px;
+    top: 10px;
+}
+
+.lado-esquerdo {
+    margin-top: 170px;
+    width: 380px;
+    height: 300px;
+    padding: 10px;
+    text-align: justify;
+    background-color: khaki;
+}
+
+.lado-direito {
+    margin-top: 170px;
+    width: 380px;
+    height: 300px;
+    padding: 10px;
+    text-align: justify;
+    background-color: khaki;
+}
+
+#rodape {
+    background-color: darkcyan;
+    height: 200px;
+    padding-top: 40px;
+    padding-left: 10px;
+    font-family: Verdana, Geneva, Tahoma, sans-serif;
+    font-size: 14px;
+    color: yellow;
```

```suggestion
    color: blue;
```
Que tal mudar esse fundo para azul? Acho que fica mais bonito!